Streamlit/requirements.txt

```diff
@@ -0,0 +1,3 @@
+streamlit
```

Comment:
that file should likely not be commited as generated by the tooling, no?

Reply:
The trick is that there are two paths to using the examples -- one is via the edge cli, which generates this file, but the _other_ is as an example checked out by itself and used with the example's ci tools directly.

If we want to remove the ability to use the examples without the edge cli, then sure, but otherwise we need this file (because the example ci doesn't generate it).

Reply:
argh, true ... that's annoying. let's keep the requirements.txt file in then.